docs/index.asciidoc

```diff
@@ -37,6 +37,25 @@ can be https://dev.maxmind.com/geoip/geoip2/geolite2[downloaded from here].
 
 If you would like to get Autonomous System Number(ASN) information, you can use the GeoLite2-ASN database.
 
+[id="plugins-{type}s-{plugin}-database_license"]
+==== Database License
+
+https://www.maxmind.com[MaxMind] changed from releasing the GeoIP database under
+a Creative Commons (CC) license to a proprietary end-user license agreement
+(EULA).  The MaxMind EULA requires Logstash to update the MaxMind database
+within 30 days of a database update. If Logstash fails to download the database
+for 30 days, it will stop the pipeline in order to maintain compliance.
```

Comment:
```suggestion
for 30 days, the geoip filter will stop enriching events in order to maintain compliance. Events will be tagged with `_geoip_expired_database` tag to facilitate the handling of these events.
```

Reply:
There will be a follow-up PR for the doc. I will update this line. thx :)